sale_contact_type/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Sale Contact Type",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
```

Comment:
This must be in the migration commit, not here.

Reply:
@LudLaf